sockeye/inference.py

```diff
@@ -21,7 +21,7 @@
 import time
 from collections import defaultdict
 from functools import lru_cache, partial
-from typing import Callable, Dict, Generator, List, NamedTuple, Optional, Tuple, Union, Set
+from typing import Callable, Dict, Generator, List, NamedTuple, Optional, Tuple, Union, Set, Any
```

Comment:
Any doesn't seem to be used

Reply:
It's used in line 894 when the dictionary `d_` is created.